editoast/src/views/timetable/similar_schedules.rs

```diff
@@ -0,0 +1,124 @@
+use axum::Extension;
+use axum::Json;
+use axum::extract::State;
+use chrono::DateTime;
+use chrono::Utc;
+use editoast_authz::Role;
+use serde::Deserialize;
+use serde::Serialize;
+use utoipa::ToSchema;
+
+use crate::error::Result;
+
+use super::AppState;
+use super::AuthenticationExt;
+use super::AuthorizationError;
+
+editoast_common::schemas! {
+    Waypoint,
+    RollingStockCharacteristics,
+    SimilarSchedulesRequest,
+    SimilarScheduleResponse,
+}
+
+crate::routes! {
+    "/similar_schedules" => {
+        similar_schedules,
+    },
+}
+
+#[derive(Debug, Deserialize, ToSchema)]
+#[allow(dead_code)]
+struct RollingStockCharacteristics {
+    name: String,
+    towed_rolling_stock: Option<String>,
+    speed_limit_tag: Option<String>,
+    mass: Option<u64>,
```

Comment:
```suggestion
    name: String,
    speed_limit_tag: Option<String>,
```

as discussed, we don't need it now since we don't have the information in timetable scheudles

Reply:
Done.